c/common/shared_dictionary_internal.h

```diff
@@ -9,9 +9,9 @@
 #ifndef BROTLI_COMMON_SHARED_DICTIONARY_INTERNAL_H_
 #define BROTLI_COMMON_SHARED_DICTIONARY_INTERNAL_H_
 
-#include "./dictionary.h"
+#include "dictionary.h"
 #include <brotli/shared_dictionary.h>
```

Comment:
can the includes be ordered by type here? first <> style includes, then "" style includes

Reply:
Those are made via copybara translation; not sure if it is possible to sort lines in a reversible manner.
However, it might be possible to modify the internal codebase, in such a manner that library includes are separate from the regular ones.